opnsense_api/interfaces/vip_controller.py

```diff
@@ -23,15 +23,15 @@ class VIPInterface(OPNsenseItem):
 class VIPController(OPNsenseItemController[VIPInterface]):
 
     def __init__(self, device):
-        super().__init__(device, "interfaces", "vip_settings")
+        super().__init__(device, "interface", "vip_settings")
 
     def _parse_api_response(self, api_response) -> VIPInterface:
         return VIPInterface(
         uuid=api_response["uuid"],
-        description=api_response["descr"],
```

Comment:
`_parse_api_response` is parsing the item JSON object sent in the devices response, which doesn't include `description`. For whatever reason it seems for that item the description key was abbreviated to `descr`

Reply:
Did we address this? I don't remember now...